plugins/com.google.cloud.tools.eclipse.test.util/src/com/google/cloud/tools/eclipse/test/util/BasePluginXmlTest.java

```diff
@@ -87,6 +98,33 @@ public final void testKeywordsDefined() {
   }
   
   @Test
+  public final void testBuildProperties() throws IOException {
+    String[] binIncludes = buildProperties.get("bin.includes").split(",\\s*");
+    Set<String> includes = Sets.newHashSet(binIncludes);
+    
+    Assert.assertTrue(includes.contains("plugin.xml"));
+    Assert.assertTrue(includes.contains("plugin.properties"));
+    Assert.assertTrue(includes.contains("."));
+    Assert.assertTrue(includes.contains("META-INF/"));
+    
+    testIncludedIfPresent(includes, "helpContexts.xml");
+    testIncludedIfPresent(includes, "icons/");
+    testIncludedIfPresent(includes, "lib/");
+    testIncludedIfPresent(includes, "README.md/");
+    testIncludedIfPresent(includes, "epl-v10.html/");
```

Comment:
yes, we can certainly expand the tests here. I think first I may want to break out testing plugin.xml from testing build.properties into separate classes.